mapsims/noise.py

```diff
@@ -195,35 +255,54 @@ def simulate(self, ch, output_units="uK_CMB", seed=None):
                     self.seed + frequency_offset + telescope_seed_offset[ch.telescope]
                 )
         zeros = np.zeros_like(self.noise_ell_T[ch.telescope][ch.center_frequency.value])
-        output_map = hp.ma(
-            np.array(
-                hp.synfast(
-                    [
-                        self.noise_ell_T[ch.telescope][ch.center_frequency.value],
-                        self.noise_ell_P[ch.telescope][ch.center_frequency.value],
-                        self.noise_ell_P[ch.telescope][ch.center_frequency.value],
-                        zeros,
-                        zeros,
-                        zeros,
-                    ],
-                    nside=self.nside,
-                    pol=True,
-                    new=True,
-                    verbose=False,
-                )
+        ps = (
+            np.asarray(
+                [
+                    self.noise_ell_T[ch.telescope][ch.center_frequency.value],
+                    self.noise_ell_P[ch.telescope][ch.center_frequency.value],
+                    self.noise_ell_P[ch.telescope][ch.center_frequency.value],
+                    zeros,
+                    zeros,
+                    zeros,
+                ]
             )
+            * nsplits
```

Comment:
does this mean that everything scales up as white noise? what about the large-scale noise coming from the atmosphere?

Reply:
That's right; both the white noise and atmosphere scaling are treated as identical. This is the noise model currently adopted in ACT, and I don't think there is evidence for it being massively wrong. @mhasself @amaurea @ajvanengelen

Reply:
@mhasself or @jcolinhill should confirm but I think linear scaling of noise power with 1/time has been assumed so far everywhere in SO noise modeling

Reply:
Yes, our red noise is assumed to average down in the usual simple way with _observing_ time.  Such red noise can be _spatially_ correlated, at any instant in time, which means that simply adding detectors (to get more "total" integration time for fixed observing time) will not necessarily help you.  ACT sees the atmosphere changing sufficiently quickly that even different optics tubes (separated by 1 deg or so) don't really suffer much correlated red noise at any scale of interest (smaller than 1 deg, say).

"Noise" that doesn't average down with observing time (all else being equal) should probably not be called noise, but rather "systematic contamination", or something like that.  We expect ground contamination to behave in that way... but we're planning on not having any of _that_...

Reply:
ok thanks @mhasself @msyriac 
@msyriac can you add a summary of this into the docstring?

Reply:
I've added a small note.